README.rst

```diff
@@ -334,10 +334,119 @@ This recipe is a little different from other recipes in that it's just a script
    COPY --from=rocky /usr/local /usr/local
    # Only needs to be run once for all recipes
    RUN for patch in /usr/local/share/just/container_build_patch/*; do "${patch}"; done
-   
+
    RUN dnf install -y --enablerepo=rocky-appstream telnet # This line is just an example
    ...
 
+CUDA
+----
+
+=========================== ============
+Name                        CUDA
+Build Args                  ``CUDA_REPO_REF`` - The version of the CUDA container repo parsed
```

Comment:
```suggestion
Build Args                  ``CUDA_REPO_REF`` - SHA of the CUDA container repo parsed
```

Reply:
Doesn't have to be a sha, it's a ref

Reply:
Then should it say The ref of the CUDA container repo

Reply:
```suggestion
Build Args                  ``CUDA_REPO_REF`` - The ref of the CUDA container repo parsed
```